tools/Lykke.Job.NeoGasDistributor.Initializer/MELogReader.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace Lykke.Job.NeoGasDistributor
+{
+    public class MELogReader
+    {
+        private readonly string _logsFolderPath;
+        
+        public MELogReader(
+            string logsFolderPath)
+        {
+            _logsFolderPath = logsFolderPath;
+        }
+        
+        public IEnumerable<MELogRecord> GetRecords()
+        {
+            var logFiles = Directory.GetFiles(_logsFolderPath, "balanceUpdates-*.json");
```

Comment:
shouldn't we order files by dates?

Reply:
Nope, the order of files does not matter